src/robusta/core/triggers/pod_crash_loop_trigger.py

```diff
@@ -48,6 +55,19 @@ def should_fire(self, event: TriggerEvent, playbook_id: str):
 
         pod = exec_event.get_pod()
 
+        run_time_seconds = 0
+
+        # startTime does not exist every time pod update is fired, like when the pod is just created
+        if pod.status.startTime:
+            run_time_seconds = (
+                datetime.utcnow().timestamp() - parse_kubernetes_datetime_to_ms(pod.status.startTime) / 1000
+            )
+
+        # sometimes pods can fire falsely on pod startup
+        # due to not loading a needed component
```

Comment:
I'm not sure what you mean in that comment.
Regardless, do we really need both checks here?
Both checking the pod is running for more than 2 minutes, and that it had more than 2 restarts?

Reply:
I think the multiple restarts would still reduce noise like we do now, 
if a pod ran for an hour and then randomly crashed but isn't looping, than since we only report on the second crash we wont report it.
I wouldnt remove it but if you would like to report every crash I would set the default restart_count to 0 and not remove it incase users want to use it to reduce noise.
WDYT?